velox/functions/lib/MapConcat.cpp

```diff
@@ -118,6 +119,13 @@ class MapConcatFunction : public exec::VectorFunction {
       for (vector_size_t i = 1; i < mapSize; i++) {
         if (combinedKeys->equalValueAt(
                 combinedKeys.get(), mapOffset + i, mapOffset + i - 1)) {
+          const auto& config = context.execCtx()->queryCtx()->queryConfig();
+          const auto sparkThrowExceptionOnDuplicateMapEntry = config.sparkThrowExceptionOnDuplicateMapEntry();
```

Comment:
It looks like we could access this config once outside the loop.

Reply:
Updated, thanks